tests/graphql/account.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { expect } from "chai";
 import * as sinon from "sinon";
 
+import { Account } from "../../lib/graphql/account";
 import { Client } from "../../lib";
 import { KontistSDKError } from "../../lib/errors";
-import { Account } from "../../lib/graphql/account";
+import { expect } from "chai";
```

Comment:
🛠 in other repositories we kept the external library at top of the file, then the internal libraries!